src/main/java/seedu/guest/model/guest/Phone.java

```diff
@@ -11,8 +11,8 @@ public class Phone {
 
 
     public static final String MESSAGE_CONSTRAINTS =
-            "Phone numbers should only contain numbers, and it should be at least 3 digits long";
-    public static final String VALIDATION_REGEX = "\\d{3,}";
+            "Phone numbers should only contain numbers. It should be at least 3 digits and at most 15 digits long.";
```

Comment:
Should we change the format for the other fields as well? or switch this back to be consistent with the others?

Reply:
I think it's fine depending on the length of the messageand the sentence structure